handlersettings.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"github.com/Azure/azure-docker-extension/pkg/vmextension"
+	"github.com/pkg/errors"
+)
+
+var (
+	errStoragePartialCredentials = errors.New("'storageAccountName' and 'storageAccountKey' must be both specified")
```

Comment:
The phrase sounds strange due to the placement of both.  I would either drop it, or move it to beginning of the sentence.